test/add-retrieve.spec.js

```diff
@@ -25,7 +25,9 @@ describe('data can be put and fetched', () => {
 
       const f = IPFSFactory.create(dfOpts)
 
-      f.spawn((err, _ipfsd) => {
+      f.spawn({
+        initOptions: { bits: 1024 }
```

Comment:
Put the initOptions as part of the tests array. This way you can do 512 for js-ipfs and 1024 for go-ipfs